qt_gui_py_common/CMakeLists.txt

```diff
@@ -1,10 +1,16 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.5)
 project(qt_gui_py_common)
 
-find_package(catkin REQUIRED)
-catkin_package()
+find_package(ament_cmake REQUIRED)
 
-catkin_python_setup()
+ament_python_install_package(${PROJECT_NAME}
+  PACKAGE_DIR src/${PROJECT_NAME})
 
 install(DIRECTORY resource
-  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
+  DESTINATION share)
```

Comment:
This should go into a package specific subdirectory as it was before.

Reply:
I'm not sure I get what you mean. Like 'share/qt_gui_py_common/resource'?

Reply:
`CATKIN_PACKAGE_SHARE_DESTINATION` expands to `share/${PROJECT_NAME}`. Keeping the same value makes sense. Otherwise installing them just to `share` might lead to collisions with other packages.

Reply:
Changing to share/${PROJECT_NAME}